src/main/java/com/agency04/sbss/pizza/PizzaApp.java

```diff
@@ -1,27 +1,13 @@
 package com.agency04.sbss.pizza;
 
-import com.agency04.sbss.pizza.model.pizzaPojos.Pizza;
-import com.agency04.sbss.pizza.service.PizzeriaService;
-import com.agency04.sbss.pizza.service.PizzaDeliveryService;
-import org.springframework.context.support.ClassPathXmlApplicationContext;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+@SpringBootApplication
 public class PizzaApp {
 
     public static void main(String[] args) throws InterruptedException {
-
-        ClassPathXmlApplicationContext context = new ClassPathXmlApplicationContext("spring-context.xml");
-        PizzaDeliveryService pizzaDeliveryService  = context.getBean("pizzaDeliveryService",PizzaDeliveryService.class);
-        Pizza pizza = context.getBean("quatroFormaggi",Pizza.class);
-
-        System.out.println(pizzaDeliveryService.orderPizza(pizza));
-
-        PizzeriaService pizzeriaService = context.getBean("dominos",PizzeriaService.class);
-        PizzeriaService pizzeriaService2 = context.getBean("pizzeriaLastrada",PizzeriaService.class);
-
-        System.out.println(pizzeriaService.getAddress());
-        System.out.println(pizzeriaService2.getName());
-
-        context.close();
+        SpringApplication.run(PizzaApp.class,args);
```

Comment:
Context mozes dobit pomocu SpringApplication.run(PizzaApp.class, args), i onda tu dohvati bean PizzaDeliveryService i ispisat vrijednost orderPizza  (mislim na ovaj dio zadatka "Make sure the app works as before, and the PizzeriaService used depends on which Bean has @Primary")